backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -2106,15 +2102,14 @@ describe('ContributionResolver', () => {
             })
           })
 
-          describe('user for creation to update is deleted', () => {
+          describe.skip('user for creation to update is deleted', () => {
```

Comment:
why skip?

Reply:
Same. It does not matter if the user exists or is deleted, as I do not query the user anymore